test/expected/inner_fun_text.txt

```diff
@@ -32,8 +32,8 @@ local children=($(find -H "$1" -maxdepth 1 -mindepth 1))
 IFS=$SAVEIFS
 local dirname=$(basename "$1")
 mkdir -p "$target/$dirname"
-for child in "${children[@]}"; do
-if [[ "$dirname" != *.ext_build_deps ]]; then
+for child in "${children[@]:-}"; do
+if [[ "$dirname" != *.ext_build_deps && "$dirname" != *.build_tempdir ]]; then
```

Comment:
This is a great addition, although I'm a little worried that we're adding these as blanket exception. One day someone will decide to name their files like this and we'll just skip them here.

It would be nice if we had the full path and passed it in to the Starlark function that generates the shell function, that would be more future proof.

But for now this is a great addition, thanks for adding it here!

Reply:
I've actually decided to drop this particular change `&& "$dirname" != *.build_tempdir` since it wasn't necessary to fix the problem I was able to reproduce